lib/gollum/views/pages.rb

```diff
@@ -37,34 +37,34 @@ def delete_file(url)
       def files_folders
         if has_results
           folders = {}
-          page_files = {}
+          files = {}
 
           # 1012: Folders and Pages need to be separated
-          @results.each do |page|
-            page_path = page.path
-            page_path = page_path.sub(/^#{Regexp.escape(@path)}\//, '') unless @path.nil?
+          @results.each do |result|
+            result_path = result.path
```

Comment:
is `result.path` escaped, and if not, is there potential for JS-insertion in line 52 below?

Reply:
Asking because in `master` we're currently sanitzing the whole HTML block outputted by this method. But it would be preferable to sanitize only the bits that are potentially user-generated.

Reply:
I don't think `result.path` is, but the outputted html uses `result.escaped_url_path`.